connector_magento_multi_source_inventory/models/magento_backend/common.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from odoo import api, models, _
+
+_logger = logging.getLogger(__name__)
+
+
+class MagentoBackend(models.Model):
+    _inherit = 'magento.backend'
+
+    @api.multi
+    def synchronize_metadata(self):
+        # super(MagentoBackend, self).synchronize_metadata()
```

Comment:
I know this is WIP but when you say tests break, not calling super is not going to help.